### PR TITLE
Feature/omit screenshots

### PR DESCRIPTION
**Pull request by roesslerj**
- [x] the code follows the [Clean Code](https://clean-code-developer.com/) guidelines.
- [x] the necessary tests are either created or updated.
- [x] all status checks (Travis CI, SonarCloud, etc.) pass.
- [x] your commit history is cleaned up.
- [ ] <s>you updated the changelog.</s>
- [ ] <s>you updated necessary documentation within [retest/docs](https://github.com/retest/docs).</s>

### CODE REVIEW

File: src/main/java/de/retest/web/screenshot/FullPageScreenshot.java [hunk 1]
```diff
@@ -11,25 +11,31 @@
...
 			final BufferedImage image = Shutterbug //
 					.shootPage( driver, ScrollStrategy.WHOLE_PAGE_CHROME, SCROLL_TIMEOUT_MS, USE_DEVICE_PIXEL_RATIO ) //
 					.getImage();
 			return resizeImage( image, image.getWidth() / SCALE, image.getHeight() / SCALE );
 		}
-		return Shutterbug //
+		final BufferedImage result = Shutterbug //
 				.shootPage( driver, ScrollStrategy.BOTH_DIRECTIONS, SCROLL_TIMEOUT_MS, USE_DEVICE_PIXEL_RATIO ) //
 				.getImage();
+		log.info( "Took {}ms to create the screenshot.", System.currentTimeMillis() - startTime );
```
Comment (diba1013) [inline src/main/java/de/retest/web/screenshot/FullPageScreenshot.java]: You do not profile in every case (i.e. `ChromeDriver`). If you really want to do profiling of the screenshots (and I currently do not see why you would), I suggest to do that in `ScreenshotProviders#shoot` in a central place and keep these implementations clean.

File: src/main/java/de/retest/web/screenshot/ViewportOnlyMinimalScreenshot.java [hunk 2]
```diff
@@ -21,7 +23,10 @@
 
 	@Override
 	public BufferedImage shoot( final WebDriver driver ) {
-		return resizeImage( Shutterbug.shootPage( driver, USE_DEVICE_PIXEL_RATIO ).getImage() );
+		final long startTime = System.currentTimeMillis();
+		final BufferedImage result = resizeImage( Shutterbug.shootPage( driver, USE_DEVICE_PIXEL_RATIO ).getImage() );
+		log.info( "Took {}ms to create the screenshot.", System.currentTimeMillis() - startTime );
+		return result;
```
Comment (diba1013) [inline src/main/java/de/retest/web/screenshot/ViewportOnlyMinimalScreenshot.java]: See my comment about profiling above

File: src/main/java/de/retest/web/screenshot/FullPageScreenshot.java (hunk 1 above)
Comment (roesslerj) [reply src/main/java/de/retest/web/screenshot/FullPageScreenshot.java]: Done

File: pom.xml [hunk 3]
```diff
@@ -238,6 +238,7 @@
 					<reportsDirectory>${project.build.directory}/surefire-reports/</reportsDirectory>
 					<systemPropertyVariables>
 						<de.retest.recheck.rehub.reportUploadEnabled>true</de.retest.recheck.rehub.reportUploadEnabled>
+						<de.retest.recheck.web.screenshot.provider>none</de.retest.recheck.web.screenshot.provider>
```
[the remark this answers is not in the document]
Comment (roesslerj) [reply pom.xml]: Done

**Pull request merged**